registration-processor/pre-processor/registration-processor-packet-classifier-stage/src/main/java/io/mosip/registration/processor/stages/packetclassifier/utility/PacketClassifierUtility.java

```diff
@@ -0,0 +1,60 @@
+package io.mosip.registration.processor.stages.packetclassifier.utility;
+
+import java.util.List;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import io.mosip.kernel.core.exception.BaseCheckedException;
+import io.mosip.registration.processor.core.exception.util.PlatformErrorMessages;
+
+@Component
+public class PacketClassifierUtility {
```

Comment:
Create a unit test class for this Utility and have 6 test case,
1 for manditory language and data is available in the first manditory language
1 for manditory language and data is not available in the first manditory language but available for 2nd manditory language
1 for optional language and data is available in the first optional language
1 for optional language and data is not available in the first optional language but available for 2nd optional language
1 for empty array in both manditory language and optional language configs
1 for data available in language that is not available in manditory or optional language

Reply:
written test cases